src/graphql/thegraph/__generated__/types-and-hooks.ts

```diff
@@ -37,6 +37,7 @@ export type Bundle = {
 export type Bundle_Filter = {
```

Comment:
This should be changed in a separate PR, right?

Reply:
i think we're supposed to check these changes in, according to @cbachmeier

Reply:
but nothing in this PR should have generated this. It's not related to this PR's changes, and should be checked in through a separate PR.

Reply:
ok, removed